v2/spanner-common/src/main/java/com/google/cloud/teleport/v2/spanner/migrations/avro/AvroToValueMapper.java

```diff
@@ -281,12 +282,12 @@ static ByteArray avroFieldToByteArray(Object recordValue, Schema fieldSchema) {
         }
         return ByteArray.copyFrom(Hex.decodeHex(s));
       }
-      return ByteArray.copyFrom((byte[]) recordValue);
+      return ByteArray.copyFrom(((ByteBuffer) recordValue).array());
     } catch (Exception e) {
       throw new AvroTypeConvertorException(
           "Unable to convert "
               + fieldSchema.getType()
-              + " to byte array, with value: "
```

Comment:
This should be array since the output type is array and not a buffer.

Reply:
Done